deploy/crds/ocmagent.managed.openshift.io_managedfleetnotificationrecords.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: managedfleetnotificationrecords.ocmagent.managed.openshift.io
+spec:
+  group: ocmagent.managed.openshift.io
+  names:
```

Comment:
Just an option additionally. ManagedFleetNotificationRecord is quite long and it will help to have a shorter name too in `CRD.spec.names.shortNames`. Something like `mfnr`?

Reply:
Yeah, that's a good point, I was thinking about the short name as well.
But I feel the shortname is more used by human, and the resource will be used by OA mostly. 
I am ok to add a shortname for both.

Reply:
Yeah, I was thinking we might still need it while testing or in rare scenario while manual troubleshooting if anything wrong with OAO on Hypershift.

Reply:
shortname for the resources added